.github/workflows/health_check.yml

```diff
@@ -0,0 +1,30 @@
+name: Health Check
+
+on:
+  # Run the workflow test on push events
+  push:
+  # Run the main workflow on workflow_dispatch or schedule
+  workflow_dispatch:
+  schedule:
+    # Every 5 minutes
+    - cron: '*/5 * * * *'
+
+jobs:
+  health_check:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        environment: ${{fromJson(github.event_name == 'push' && '["local"]' || '["dev","stage","prod"]')}}
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: ./.github/actions/run-docker
```

Comment:
Needing the entire docker setup just to run a basic python script that doesn't have any dependencies other than `requests` feels like a waste of resources and is quite slow (even with docker build caching).

Reply:
I hear you, but this way it ensures that we are running the same environment as when you run this locally. Otherwise we have to introduce a totally separate way of loading dependencies and I don't think that's worth the cost of maintaining it when we have a ready to go solution for "running stuff" wdyt?

Reply:
Random observation: AFAICT we don't allow concurrent workflow runs so the slowness could have an impact, if one run takes more than 5 minutes, that might delay the others... And the recent ones all took over 4 minutes...

Reply:
This workflow doesn't have concurrency rules defined so it should be fine.

Reply:
I think there are probably areas where we could make it more efficient. Could for example pull the image instead of building or instead of using a matrix job could iterate on the script level in a single job. I'd put those in a follow up after we get the slack notification working and let it run for a week or so to see what actually makes the most sense.